packages/test-app/src/App.vue

```diff
@@ -1,20 +1,26 @@
 <template>
-	<div id="app">
-		<KtForm v-model="values">
-			<KtTranslationContext locale="en-US">
-				<KtFieldText formKey="text" label="Text" />
-				<KtFieldDate formKey="date" label="Date"
-			/></KtTranslationContext>
-		</KtForm>
-	</div>
+	<KtTranslationContext locale="en-US">
+		<div style="padding: 50px;">
+			<KtAccordion title="Hello Tree Shaking">
+				<KtForm v-model="values">
+					<KtHeading text="Test App" />
+					<KtFieldText formKey="text" label="Text" />
+					<!-- <KtFieldDate formKey="date" label="Date" /> -->
```

Comment:
@FlorianWendelborn do we remove this comment

Reply:
no, i think they make sense to keep. in the future even to still test that the tree shaking still works (assuming config changes), uncommenting those lines should be the easiest workflow, you were right about that.